composer.json

```diff
@@ -53,5 +54,8 @@
         }
     },
 
+    "prefer-stable": true,
+    "minimum-stability": "dev",
```

Comment:
I feel uneasy about doing this with stable core release. If that's the only way of doing the feature, my preference shifts to just providing an extension point in the core, but Puli being an extension (maybe even official).